index.html

```diff
@@ -1,12 +1,66 @@
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <script src="js/index.js"></script>
+  <header>
+    <h1 id="title">FreeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+  </header>
+  <form id="survey-form" action="https://survey-form.freecodecamp.rocks/">
+    <div id="container">
+      <p>Name</p>
+      <input id="name" type="text" placeholder="Enter your name" required>
+      <p>Email</p>
+      <input id="email" type="email" placeholder="Enter your email" required>
+      <p>Age (optional)</p>
+      <input id="number" type="number" min="10" max="99" required>
+      <p>Which option describes your current role?</p>
+      <select name="" id="">
+        <option name="first" value="0">Select current role</option>
+        <option name="first" value="1">Student</option>
+        <option value="2">Full Time Job</option>
+        <option value="3">Full Time Leaner</option>
+        <option value="4">Prefer not to say</option>
+        <option value="5">Other</option>
+      </select>
+      <p>Would you recommend freeCodeCamp to a friend?</p>
+      <label for="definately"> <input type="radio" name="recommend" value="recommend" required> Definately</label>
+      <label for="maybe"> <input type="radio" name="recommend" value="recommend" required> Maybe</label>
+      <label for="not sure"> <input type="radio" name="recommend" value="recommend" required> Not sure</label>
+      <p>What is your favorite feature of freeCodeCamp?</p>
+      <select name="" id="">
+        <option value="">Select an option</option>
+        <option value="1">Projects</option>
+        <option value="2">Challanges</option>
+        <option value="3">Community</option>
+        <option value="4">Open Source</option>
+      </select>
+      <p>What would you like to see improved? (Check all that apply)</p>
+      <label for=""> <input type="checkbox"> Front-end Project </label>
+      <label for=""><input type="checkbox"> Back-end Projects</label>
+      <label for=""><input type="checkbox"> Data Visualization</label>
+      <label for=""><input type="checkbox"> Challenges</label>
+      <label for=""><input type="checkbox"> Open Source Community</label>
+      <label for=""><input type="checkbox"> Gitter help rooms</label>
+      <label for=""><input type="checkbox"> Videos</label>
+      <label for=""><input type="checkbox"> City Meetups</label>
+      <label for=""><input type="checkbox"> Wiki</label>
+      <label for=""><input type="checkbox"> Forum</label>
+      <label for=""><input type="checkbox"> Additional Courses</label>
```

Comment:
The for attributes are declared with no values. It is important to do so as your markUp would be regarded as an invalid HTML. Kindly update it to ensure a functional and valid html.